src/ini_to_core_reader.cpp

```diff
@@ -15,16 +15,23 @@
 #include <boost/lexical_cast.hpp>
 #include <boost/algorithm/string/predicate.hpp>
 #include <boost/algorithm/string/trim.hpp>
+
+// ANS: If using the R package, use Rcpp to call R's file processing
```

Comment:
Yes, "Alexey Nikolaevich Shiklomanov", and happy to remove these.

That said, in my experience, I don't find `git blame` (and `git log <filename>`) to be 100% reliable. For one, when files get moved without `git mv` (a common mistake) and then heavily edited -- a frequent combination for a major refactor -- the git history for individual files can get lost. On the other hand, you can get false positives when comments get reformatted -- for instance, @bpbond is more-or-less the default `git blame` target of a bunch of the PEcAn code base because he ran an automatic code formatter on all of our code, even almost none of those changes were functional. Anyway, just my 2 cents...